Console/Command/RestoreUseDefaultValueCommand.php

```diff
@@ -140,9 +215,12 @@ public function execute(InputInterface $input, OutputInterface $output): int
             if (!$isDryRun && $nullCount > 0) {
                 $output->writeln("Deleting $nullCount NULL value(s) from $fullTableName");
                 // Remove all non-global null values
-                $dbWrite->query(
-                    'DELETE FROM ' . $fullTableName . ' WHERE store_id != 0 AND value IS NULL'
+                $removeNullValuesQuery = sprintf('DELETE FROM ' . $fullTableName . ' WHERE store_id != 0 %s %s AND value IS NULL',
+                    $storeIdFilter,
+                    $attributeFilter
                 );
+                $output->writeln(sprintf('<info>%s</info>', $removeNullValuesQuery));
```

Comment:
Do we output queries also in other commands? I'm not fully against it, I just think it should be consistent :)

Reply:
No we don't AFAIK, but I think it really helps to understand what is happening under the hood
Just let me know if you want me to remove them or also add them to the other commands

Reply:
I don't know :joy: I'm just a fan of consistency, but all fine :)